tests/visco_plastic_phases_plasticity.prm

```diff
@@ -0,0 +1,105 @@
+# Copy of visco_plastic_phases.prm, but tests different plasticity parameters for different phases.
```

Comment:
Can you add some additional comments here describing what happens (e.g., how values change above and below phase transition) and what files this should be visible in (e.g., ascii visualization output)?